packages/mds-attachment-service/package.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "build": "tsc --build tsconfig.build.json",
     "start": "pnpm watch server",
-    "pretest": "pnpm typeorm -- schema:drop",
+    "pretest": "env $(cat ../../.env | xargs) pnpm typeorm -- schema:drop",
```

Comment:
Have these changes been verified on `Windows`?

Reply:
Could "typeorm" just call "pnpm ts-node" instead of running "ts-node" directly?

Reply:
argh, `Windows`? What's that?

Reply:
> Could "typeorm" just call "pnpm ts-node" instead of running "ts-node" directly?

what do you mean with that?

Reply:
> > Could "typeorm" just call "pnpm ts-node" instead of running "ts-node" directly?
> 
> what do you mean with that?

Could you change:
```
"typeorm": "ts-node ./node_modules/typeorm/cli.js"
```
to:
```
"typeorm": "pnpm ts-node ./node_modules/typeorm/cli.js"
```
Does that work? Allowing "pretest" to remain as it was?

And `Windows`, yeah, there are plenty of folks that still use it so we have to ensure our tooling works (especially in an open source repository).

Reply:
I have used `cross-env` in the past. I am not familiar with `env-cmd`, but I am not opposed if it works. Sounds like it tries to solve similar issues in a cross platform way.

Reply:
I tested it locally and it met the expectations. Couldn't verify on `Windows` though.
When adding the dependency, the `pnpm-lock.yaml` oinned tarballs from our corporate registry (although I commented it in the `.npmrc`. Therefore, I didn't commit it. May somebody please let me know how to re-install from npmjs.org or add the file?

Reply:
@mrsimpson Perhaps a `pnpm clean`? That should regenerate the `pnpm-lock.yaml`.

Reply:
Yes, I thought so too and did that already. But it still picks up the local registry. 
pnpm is still a bit of a mystery to me...

Reply:
I used another machine to re-install the packages. Thus, this part looks ok for me now